interfaces/builtin/online_accounts_service.go

```diff
@@ -0,0 +1,186 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var onlineAccountsServicePermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the Online Accounts service. Reserved because
+# this gives privileged access to the system.
+
+# DBus accesses
+#include <abstractions/dbus-session-strict>
+
+dbus (send)
+    bus=session
+    path=/org/freedesktop/DBus
+    interface=org.freedesktop.DBus
+    member={RequestName,ReleaseName,GetConnectionCredentials}
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+# Allow binding the service to the requested connection name
+dbus (bind)
+	bus=session
+	name="com.ubuntu.OnlineAccounts.Manager",
+`)
+
+var onlineAccountsServiceConnectedSlotAppArmor = []byte(`
+# Allow service to interact with connected clients
+dbus (receive, send)
+	bus=session
+	path=/com/ubuntu/OnlineAccounts{,/**}
+	interface=com.ubuntu.OnlineAccounts.Manager
+	peer=(label=###PLUG_SECURITY_TAGS###),
+`)
+
+var onlineAccountsServiceConnectedPlugAppArmor = []byte(`
+# Description: Allow using Online Accounts service. Common because the access
+# to user data is actually mediated by the Online Accounts service itself.
+# Usage: common
```

Comment:
Please drop 'Usage' metadata here and elsewhere. It is no longer needed with the base declaration.